app/Resources/views/substitute/index.html.twig

```diff
@@ -94,6 +97,18 @@
                             <td>{{ sub.wednesday }}</td>
                             <td>{{ sub.thursday }}</td>
                             <td>{{ sub.friday }}</td>
+                            <td>{{ sub.preferredGroup }}</td>
+                            {% if sub.interview is not null %}
+                                <td>{{ sub.interview.interviewScore.getSum() }}</td>
```

Comment:
I intervjutabellen er intervjuscoren skjult for teammedlemmer.
Kun teamledere og personen som gjennomførte intervjuet har lov til å se score og suitableAssistant.
Ingen har lov til å se score på sitt eget intervju

I intervjutabellen er disse skjult slik:
```twig
{% if a.user != app.user and ( is_granted_team_leader() or a.interview.interviewer == app.user ) %}
    <td>{{ a.interview.interviewScore.sum }}</td>
    <td>{{ a.interview.interviewScore.suitableAssistant }}</td>
    <td>{% if a.interview.application.specialNeeds %}
            <i class="fa fa-2x fa-exclamation-triangle text-warning"></i>

        {% endif %}</td>
    <td>
        <a href="{{ path('interview_show', { 'id': a.id }) }}" class="button tiny"><i
                    class="fa fa-list-ul"></i> Les intervju</a>
    </td>
{% else %}
    <td colspan="4">
        <span class="text-alert">Intervjuet er skjult</span>
    </td>
{% endif %}
```